### PR TITLE
Update changelog for 2.2 release

### PR DESCRIPTION
**Pull request by XWB**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,9 +1,21 @@
 Changelog
 =========
 
-### 2.1.3 (2018-xx-xx)
+### 2.2.0 (2020-01-08)
 
 * Added Mongolian translation.
+* Added an email prodiver.
```
Comment (stof) [inline Changelog.md]: Typo.

File: Changelog.md [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
+* Added a custom user checker.
+* Added PHP 7.3 support.
+* Removed fieldName attribute in MongoDB mapping.
+* Registration confirmation now redirects to login page if token is invalid.
+* Bumped default minimal password length to 8 characters.
```
Comment (stof) [inline Changelog.md]: this is a BC break for existing projects. I think we need to revert that commit and re-apply it only in 3.x

File: Changelog.md [hunk 3]
```diff
@@ -1,9 +1,21 @@
 Changelog
 =========
 
-### 2.1.3 (2018-xx-xx)
+### 2.2.0 (2020-01-08)
 
 * Added Mongolian translation.
+* Added an email prodiver.
+* Added a custom user checker.
+* Added PHP 7.3 support.
```
Comment (stof) [inline Changelog.md]: Could we provide PHP 7.4 support too ?

File: Changelog.md (hunk 2 above)
Comment (XWB) [reply Changelog.md]: Good catch: https://github.com/FriendsOfSymfony/FOSUserBundle/pull/2954

File: Changelog.md (hunk 3 above)
Comment (XWB) [reply Changelog.md]: https://github.com/FriendsOfSymfony/FOSUserBundle/pull/2955

**Comment by stof** [comment]
One thing to do before the release would be to make the CI green again.

**Comment by XWB** [comment]
More PR's are coming to fix the tests. They have been broken for ages.

**Pull request merged**